### PR TITLE
Document the missing website permission

### PR DESCRIPTION
**Pull request by pankajastro**
---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Review by vincbeck — APPROVED** [review]

File: docs/apache-airflow-providers-fab/auth-manager/access-control.rst [hunk 1]
```diff
@@ -228,19 +241,50 @@ Toggle DAG paused status               DAGs.can_edit
...
 Set Task Instance as up_for_retry      DAGs.can_edit                                                           User
 Autocomplete                           DAGs.can_read                                                           Viewer
+Show Dataset menu                      Datasets.menu_access                                                    Viewer
+Show Datasets                          Datasets.can_read                                                       Viewer
+Show Docs menu                         Docs.menu_access                                                        Viewer
+Show Documentation menu                Documentation.menu_access                                               Viewer
+Show Jobs menu                         Jobs.menu_access                                                        Viewer
+Show Audit Log                         Audit Logs.menu_access                                                  Viewer
+Reset Password                         My Password.can_read, My Password.can_edit                              Viewer                      Viewer
```
Comment (aliotta) [inline docs/apache-airflow-providers-fab/auth-manager/access-control.rst]: I think you have one too many "Viewer" here

File: docs/apache-airflow-providers-fab/auth-manager/access-control.rst [hunk 2]
```diff
@@ -196,19 +200,28 @@ Website Permissions
...
 Access homepage                        Website.can_read                                                        Viewer
+Show Browse menu                       Browse.menu_access                                                      Viewer
+Show DAGs menu                         DAGs.menu_access                                                        Viewer
 Get DAG stats                          DAGs.can_read, DAG Runs.can_read                                        Viewer
+Show Task Instances menu               Task Instances.menu_access                                              Viewer
 Get Task stats                         DAGs.can_read, DAG Runs.can_read, Task Instances.can_read               Viewer
 Get last DAG runs                      DAGs.can_read, DAG Runs.can_read                                        Viewer
 Get DAG code                           DAGs.can_read, DAG Code.can_read                                        Viewer
 Get DAG details                        DAGs.can_read, DAG Runs.can_read                                        Viewer
+Show DAG Dependencies menu             DAG Dependencies menu                                                   Viewer
```
Comment (aliotta) [inline docs/apache-airflow-providers-fab/auth-manager/access-control.rst]: should the second column be DAG Dependencies.menu_access

File: docs/apache-airflow-providers-fab/auth-manager/access-control.rst [hunk 3]
```diff
@@ -196,19 +200,28 @@ Website Permissions
...
+Get DAG Dependencies                   DAG Dependencies.can_read                                               Viewer
 Get rendered DAG                       DAGs.can_read, Task Instances.can_read                                  Viewer
 Get Logs with metadata                 DAGs.can_read, Task Instances.can_read, Task Logs.can_read              Viewer
 Get Log                                DAGs.can_read, Task Instances.can_read, Task Logs.can_read              Viewer
 Redirect to external Log               DAGs.can_read, Task Instances.can_read, Task Logs.can_read              Viewer
 Get Task                               DAGs.can_read, Task Instances.can_read                                  Viewer
+Show XCom menu                         XComs.menu_access                                                       Op
 Get XCom                               DAGs.can_read, Task Instances.can_read, XComs.can_read                  Viewer
+Create XCom                            XComs.can_create                                                        Op #TODO
```
Comment (aliotta) [inline docs/apache-airflow-providers-fab/auth-manager/access-control.rst]: whats the TODO here?

Comment (pankajastro) [reply docs/apache-airflow-providers-fab/auth-manager/access-control.rst]: removed. this was unintended. thanks for pointing out

**Pull request merged**